SportAlert.php

```diff
@@ -1,103 +1,121 @@
+					$tName = $userSettingData["TeamName"];
+					if(teamNameInTable($tName)) {
+						$array = getAllGamesForTeam($tName);
+						if($array[0]["TeamName"] != "") {
+						    $message = $message."Today there is a ".$array[0]["TeamName"]." game in ".$array[0]["Location"]."!\n";
+						 	$message = $message."Be sure to tune in for the game at ".$array[0]["Time"].".\n";
+						}
+					}
 
-		if(mysqli_connect_error()) {
-			die("ERROR: Could not connect to database<br/>");
-		}
-		print("Connection Successful - DB_SportAlert<br/><br/>");
-	}
-	
-	/*
-		Given a team, search DB to see if valid
-	*/
-	function teamNameInTable($tName) {
-		global $link;
-		$query = "SELECT * FROM `Teams`";
-
-		if($result = mysqli_query($link, $query)) {
-			while($row = mysqli_fetch_array($result)) {
-				// print($row["TeamName"]);
-				if($row["TeamName"] == $tName) {
-					// print("Found My Team: ".$tName."<br/>");
-					return true;
 				}
 			}
-		}
-		else {
-			die("ERROR: teamNameInTable() query did not work<br/>");
-		}
-		return false;
-	}
-
-	/*
-		Given a team, get all games played by said team
-	*/
-	// TODO: explore alternative method of using 'join'
-	// $query ="SELECT * FROM `Games` inner join Teams on Games.TeamName = Teams.TeamName";
-	function getAllGamesForTeam($tName) {
-		global $link;
-		$query = "SELECT * FROM `Games` WHERE `TeamName`='".$tName."'";
-
-		if($result = mysqli_query($link, $query)) {
-			while($row = mysqli_fetch_array($result)) {
-				$array[] = $row;
+			else {
+				die("ERROR: unable to access user settings db<br/>");
 			}
-			// print_r($array);
-		}
-		else {
-			die("ERROR: getAllGamesForTeam() query did not work<br/>");
-		}
-		return $array;
-	}
 
-	/*
-		Given an array of games, find games for today
-	*/
-	function gamesToday($array) {
-		$date = date('Y-m-d');
-		for($count = 0; $count < sizeof($array); $count++) {
-			if($array[$count]["Date"] == $date) {
-				$array[] = $array[$count];
-			}
+			// # Send Notification
+			// TODO: ENABLE BELOW - production purposes
+			// if( mail ( $to , $subject , $message)) {
+			// 	print "Successfully sent email";
+			// }
+			// else {
+			// 	die("ERROR: unable to send email");
+			// }
+			// TODO: DISABLE BELOW - only for test purposes
+			print($to."<br/>");
+			print($message."<br/>");
 		}
-		return $array;
 	}
-
-	/*
-		Given an array of games, find games that are upcomming
-	*/
-	function gamesAfterToday($array) {
-		$date = date('Y-m-d');
-		for($count = 0; $count < sizeof($array); $count++) {
-			if($array[$count]["Date"] > $date) {
-				$array[] = $array[$count];
-			}
-		}
-		return $array;
+	else {
+		die("ERROR: unable to access user db<br/>");
 	}
 
-	/*
-		Main
-	*/
-	makeConnection();
-	teamNameInTable($myTeam);
-	$array = getAllGamesForTeam($myTeam);
-	$array = gamesToday($array);
-	print("Today there is a ".$array[0]["TeamName"]." game in ".$array[0]["Location"]."!<br/>");
-	print("Be sure to tune in for the game at ".$array[0]["Time"].".<br/>");
+?>
```

Comment:
Extra space is not needed...was introduced to enable scroll down during development. Editor preference band-aid.